docs/develop/rust/overview.mdx

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 1
+description: "Get an idea of what a Restate Rust service looks like."
+---
+
+import Admonition from '@theme/Admonition';
+
+# Overview
+
+The Restate Rust SDK is open source and can be found on GitHub: ([sdk-python repo](https://github.com/restatedev/sdk-rust)).
+
+<Admonition type="info" icon="🚧" title="Work in progress">
+The Rust SDK documentation is still under construction. Please refer to the [restate-sdk crate](https://docs.rs/restate-sdk/latest/restate_sdk/) to get started.
```

Comment:
Add a link to the hello world example too: https://github.com/restatedev/examples/tree/main/templates/rust

Reply:
Done!